models/zeroex/bnb/zeroex_bnb_api_fills.sql

```diff
@@ -317,9 +319,18 @@ uni_v2_swap as (
         {% else %}
         AND block_time >= TIMESTAMP '{{zeroex_v4_start_date}}'
         {% endif %}
-         
     
-)  
+)
+, uni_v2_swap as (
+    SELECT *
+        ,CASE WHEN amount0Out = UINT256 '0'
+            THEN amount1Out ELSE amount0Out
+        END AS maker_token_amount_raw
+        ,CASE WHEN amount0In = UINT256 '0' OR amount1Out = UINT256 '0'
+            THEN amount1In ELSE amount0In
+        END AS taker_token_amount_raw
+    FROM uni_v2_swap_raw
+)
```

Comment:
I tried a new implementation following `uniswap_v2_ethereum_trades` to see if this fix the issue.